include/keepkey/firmware/coins.def

```diff
@@ -17,7 +17,7 @@ X(true, "EOS",          true, "EOS",   false, NA,    false,         0,  false, N
 X(true, "Axe",          true, "AXE",   true,  55,    true,     100000,  true,  16,  true, "\x19" "DarkCoin Signed Message:\n",      true, 0x80001092,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 50221772,  true, false,  true, false,  true, SECP256K1_STRING,  false, "",            false, "",     false, false,  false,        0,        false,       0)
 X(true, "Bitcore",      true, "BTX",   true,   3,    true,     100000,  true,  125, true, "\x18" "BitCore Signed Message:\n",       true, 0x800000a0,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, true,   true, false,  true, SECP256K1_STRING,  false, "",            true, "btx",   false, false,  true,  77429938,        true, 78792518)
 X(true, "Fujicoin",     true, "FJC",   true,  36,    true,   10000000,  true,  16,  true, "\x19" "FujiCoin Signed Message:\n",      true, 0x8000004b,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, true,   true, false,  true, SECP256K1_STRING,  false, "",            true, "fc",    false, false,  true,  77429938,        true, 78792518)
-X(true, "Denarius",     true, "DNR",   true,  30,    true,     100000,  true,  90,  true, "\x19" "Denarius Signed Message:\n",      true, 0x80000074,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,  false, "",            false, "",     false, false,  false,        0,        false,       0)
+X(true, "Denarius",     true, "D",   true,  30,    true,     100000,  true,  90,  true, "\x19" "Denarius Signed Message:\n",      true, 0x80000074,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,  false, "",            false, "",     false, false,  false,        0,        false,       0)
```

Comment:
```suggestion
X(true, "Denarius",     true, "D",     true,  30,    true,     100000,  true,  90,  true, "\x19" "Denarius Signed Message:\n",      true, 0x80000074,   false,   0,     true,   8,  false, NO_CONTRACT,      true, 76067358,  true, false,  true, false,  true, SECP256K1_STRING,  false, "",            false, "",     false, false,  false,        0,        false,       0)
```

Reply:
(aligns the columns in the table)

Reply:
Thanks for that, anymore insight on when it will actually be added to the Keepkey Client? Any other development that needs done?

Reply:
I should set expectations here very low: having firmware support for a coin does not imply we'll be adding it to the KeepKey Client.

Best way to get it supported independent of that is to add KeepKey support via keepkey.js to some client that already supports Denarius.

Reply:
Ah, gotcha, well that is kind of a bummer. There is nothing that we can do as a community to attempt to get it in the Keepkey Client? I am assuming it is closed sourced?

Reply:
Also is Denarius available in the current stable firmware build for the Keepkey then?

Reply:
It's in the beta firmware.

Reply:
Okay thanks, so keepkey.js has no client examples just the basic interfacing with the keepkey examples? Thanks for all the help!

Reply:
MEW is the best open source example at the moment: https://github.com/MyEtherWallet/MyEtherWallet

There's also a simple xpub dumper here: https://github.com/keepkey/keepkey.js/pull/24#issuecomment-460348006

Reply:
Yea, one will just hope we are just added to your Keepkey Client at some point...I would be more than happy to commit development for it needed...